packages/ckeditor5-dev-release-tools/tests/release/updateversions.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const expect = require( 'chai' ).expect;
+const sinon = require( 'sinon' );
+const proxyquire = require( 'proxyquire' );
+
+describe( 'dev-release-tools/release', () => {
+	let updateVersions, sandbox, outputJsonSyncStub, getPackageJsonStub, syncStub, shExecStub, gtStub;
+
+	describe( 'updateVersions()', () => {
+		beforeEach( () => {
+			sandbox = sinon.createSandbox();
+
+			outputJsonSyncStub = sandbox.stub();
+			getPackageJsonStub = sandbox.stub().returns( { version: '1.0.0' } );
+			syncStub = sandbox.stub().returns( [ '/ckeditor5-dev' ] );
+			shExecStub = sandbox.stub().throws( new Error( 'is not in this registry' ) );
+			gtStub = sandbox.stub().returns( true );
+
+			updateVersions = proxyquire( '../../lib/release/updateversions.js', {
+				'fs-extra': { outputJsonSync: outputJsonSyncStub },
+				'../utils/getpackagejson': getPackageJsonStub,
+				'glob': { globSync: syncStub },
+				'@ckeditor/ckeditor5-dev-utils': { tools: { shExec: shExecStub } },
+				'semver': { gt: gtStub }
+			} );
+		} );
+
+		afterEach( () => {
+			sandbox.restore();
+		} );
+
+		it( 'should call outputJsonSyncStub 4 times when the version is correct and there are 3 packages and 1 root', () => {
+			syncStub.returns( [
+				'/ckeditor5-dev/packages/package1',
+				'/ckeditor5-dev/packages/package2',
+				'/ckeditor5-dev/packages/package3',
+				'/ckeditor5-dev'
+			] );
+
+			updateVersions( { version: '1.0.1', packagesDirectory: 'packages' } );
+
+			expect( syncStub.firstCall.args[ 0 ] ).to.deep.equal( [ 'packages/*', './' ] );
+			expect( outputJsonSyncStub.callCount ).to.equal( 4 );
+			expect( outputJsonSyncStub.firstCall.args[ 0 ] ).to.equal( '/ckeditor5-dev/packages/package1/package.json' );
+			expect( outputJsonSyncStub.firstCall.args[ 1 ] ).to.deep.equal( { version: '1.0.1' } );
+		} );
+
+		it( 'should call outputJsonSyncStub 1 time when the version is correct packages are undefined and 1 root', () => {
+			syncStub.returns( [ '/ckeditor5-dev' ] );
+
+			updateVersions( { version: '1.0.1' } );
+
+			expect( syncStub.firstCall.args[ 0 ] ).to.equal( './' );
+			expect( outputJsonSyncStub.callCount ).to.equal( 1 );
+			expect( outputJsonSyncStub.firstCall.args[ 0 ] ).to.equal( '/ckeditor5-dev/package.json' );
```

Comment:
Not true on Windows, where instead of slashes there are backslashes: `\ckeditor5-dev\package.json`.